### DISCUSSION TITLE
Images outside project folder

### DISCUSSION THREAD

**Discussion by zodiac-test** [squidfunk/mkdocs-material#6203] [Q&A] [answered]
Is there a way to link the images in a markdown file, if the images are located outside the main project folder? For instance,

-images_folder/
----- img1.png
-project/
-----mkdocs.yml
-----docs/
----------mdfile1.md

How can I use img1 inside mdfile1? Is there any way?

**Comment by alexvoss** [comment, marked as answer]
could you use symbolic links to 'import' your images folder into the `docs` folder?

**Reply by zodiac-test** [reply]
Hi, thanks for the suggestion. Yes I could, and this works too.

**Reply by zodiac-test** [reply]
For the others, who are facing the same issue, you can create a symlink like this:

1. Open CMD as admin and cd to the destination folder.
2. Create a symlink like this: `mklink /D LinkName C:\path\to\source\folder`. An example: `mklink /D ToolsLink C:\Users\abc\def\source_folder`
3. You can now access the "imported" folder in your destination folder as if it were a regular subfolder.
